### DISCUSSION TITLE
DirectTV Stream button

### DISCUSSION THREAD

**Discussion by sjbelisle** [PRProd/HA-Firemote#203] [General]
This is a great card and I have buttons for Prime Video, NETFLIX, Paramount + and YouTube working well. Unfortunately when a configure a button for DirectTV Stream there is no response when it is pressed. I've tried this in various button positions, all with the same results. Any thoughts on what I should try next?

**Comment by PRProd** [comment]
@sjbelisle 

Thanks!  I'm glad that you are enjoying my card!

To help answer your question, I'm going to need a little more information from you.  First, what kind of device are you controlling?
(choose one)
 - Amazon Fire
 - Chromecast
 - NVIDIA Shield
 - Xiaomi Mi

**Reply by sjbelisle** [reply]
I'm controlling an Amazon Fire Stick

**Comment by PRProd** [comment]
@sjbelisle - It looks like the authors of that App have made some changes recently, so I apologize that it wasn't caught sooner.  I'll have a fix for this published soon.  Thanks for letting me know!

**Comment by sjbelisle** [comment]
Thanks for the great work on this

**Comment by PRProd** [comment]
@sjbelisle - This has been fixed in the latest release version[ v3.0.0](https://github.com/PRProd/HA-Firemote/releases/tag/v3.0.0)

Thanks again for letting me know about it!
